tool/tsh/mfa.go

```diff
@@ -669,7 +669,7 @@ func showOTPQRCode(k *otp.Key) (cleanup func(), retErr error) {
 	}
 	log.Debugf("Opened QR code via %q", imageViewer)
 	return func() {
-		if err := os.Remove(imageFile.Name()); err != nil {
+		if err := utils.RemoveSecure(imageFile.Name(), 3); err != nil {
```

Comment:
```suggestion
		const iterations = 3
		if err := utils.RemoveSecure(imageFile.Name(), iterations); err != nil {
```

(Although I'd rather drop the param altogether.)